templates/service-template.yml

```diff
@@ -540,6 +540,16 @@ parameters:
   description: kas-fleetshard operator OLM package name
   value: "kas-fleetshard-operator"
 
+- name: OBSERVABILITY_OPERATOR_INDEX_IMAGE
+  displayName: Observability operator index image
+  description: Observability operator index image
+  value: "quay.io/rhoas/observability-operator-index:v3.0.14"
+
+- name: OBSERVABILITY_OPERATOR_STARTING_CSV
+  displayName: Observability operator subscription starting csv
+  description: Observability operator subscription starting csv
+  value: "observability-operator.v3.0.14"
```

Comment:
Once this is merged, can we update app-interface to include the new parameters?

Reply:
New default version (v3.0.14) should be picked and Operator should upgrade. I can update app-interface after this is merged